content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -154,3 +156,226 @@ Follow the instructions available in CSM Authorization for [Configuring a Dell C
 **Authorization v2.0 Technical Preview**
 
 Follow the instructions available in CSM Authorization for [Configuring PowerFlex with Authorization](../../../../authorization/v2.0-tech-preview/configuration/powerflex).
+
+## Vault Server Installation
```

Comment:
Should this also be added to the helm section? Or we could have it in one page and link to it in both locations.